src/tlo/methods/cervical_cancer.py

```diff
@@ -0,0 +1,1802 @@
+            Types.LIST,
+            "initial proportions in hpv cancer categories in women without hiv"
+        ),
+        "r_hpv": Parameter(
+            Types.REAL,
+            "probability per month of oncogenic hpv infection",
+        ),
+        "r_cin1_hpv": Parameter(
+            Types.REAL,
+            "probability per month of incident cin1 amongst people with hpv",
+        ),
+        "prob_revert_from_cin1": Parameter(
+            Types.REAL,
+            "probability of reverting from cin1 to none",
+        ),
+        "r_cin2_cin1": Parameter(
+            Types.REAL,
+            "probability per month of incident cin2 amongst people with cin1",
+        ),
+        "r_cin3_cin2": Parameter(
+            Types.REAL,
+            "probability per month of incident cin3 amongst people with cin2",
+        ),
+        "r_stage1_cin3": Parameter(
+            Types.REAL,
+            "probability per month of incident stage1 cervical cancer amongst people with cin3",
+        ),
+        "r_stage2a_stage1": Parameter(
+            Types.REAL,
+            "probability per month of incident stage2a cervical cancer amongst people with stage1",
+        ),
+        "r_stage2b_stage2a": Parameter(
+            Types.REAL,
+            "probability per month of incident stage2b cervical cancer amongst people with stage2a",
+        ),
+        "r_stage3_stage2b": Parameter(
+            Types.REAL,
+            "probability per month of incident stage3 cervical cancer amongst people with stage2b",
+        ),
+        "r_stage4_stage3": Parameter(
+            Types.REAL,
+            "probability per month of incident stage4 cervical cancer amongst people with stage3",
+        ),
+        "rr_progress_cc_hiv": Parameter(
+            Types.REAL, "rate ratio for progressing through cin and cervical cancer stages if have unsuppressed hiv"
+        ),
+        "rr_hpv_vaccinated": Parameter(
+            Types.REAL,
+            "rate ratio for hpv if vaccinated - this is combined effect of probability the hpv is "
+            "vaccine-preventable and vaccine efficacy against vaccine-preventable hpv ",
+        ),
+        "rr_hpv_age50plus": Parameter(
+            Types.REAL,
+            "rate ratio for hpv if age 50 plus"
+        ),
+        "prob_cure_stage1": Parameter(
+            Types.REAL,
+            "probability of cure if treated in stage 1 cervical cancer",
+        ),
+        "prob_cure_stage2a": Parameter(
+            Types.REAL,
+            "probability of cure if treated in stage 2a cervical cancer",
+        ),
+        "prob_cure_stage2b": Parameter(
+            Types.REAL,
+            "probability of cure if treated in stage 2b cervical cancer",
+        ),
+        "prob_cure_stage3": Parameter(
+            Types.REAL,
+            "probability of cure if treated in stage 3 cervical cancer",
+        ),
+        "r_death_cervical_cancer": Parameter(
```

Comment:
would be better to label this `r_death_stage4` to match the convention established in other parameter names, and to make it clear that death only occurs for those in stage4